src/trumbowyg.js

```diff
@@ -1914,6 +1914,7 @@ Object.defineProperty(jQuery.trumbowyg, 'defaultOptions', {
             t.clearButtonPaneStatus();
 
             $.each(tags, function (i, tag) {
+                if (typeof tag === "undefined") return; // in the unlikely event that the tag is not defined skip it as to not raise an error
```

Comment:
I think this should be fixed here instead: 
https://github.com/urosg80/Trumbowyg/blob/d2a066b9eac68f8a34513e620373a28696ae14c2/src/trumbowyg.js#L1975

Reply:
Seems so yes, but since I had this fixed present in version 2.21.0 which I used before maybe it is not pertinent anymore so I will first test and review without fix in the latest version and report back.